astro_2.0/features/CreateProposal/components/CreateProposalCard/CreateProposalCard.tsx

```diff
@@ -132,25 +131,9 @@ export const CreateProposalCard: React.FC<CreateProposalCardProps> = ({
 
   function renderProposer() {
     switch (type) {
-      case ProposalVariant.ProposeContractAcceptance:
       case ProposalVariant.ProposeCreateToken: {
         return null;
       }
-      case ProposalVariant.ProposeTokenDistribution: {
```

Comment:
Why do you remove this block? By design we have to render this content

Reply:
@VERZUOL1 the `renderProposer` method is being called in a single place, in the `renderCardContent` method.
The `renderCardContent` has following content
```
function renderCardContent() {
    switch (type) {
      case ProposalVariant.ProposeContractAcceptance:
      case ProposalVariant.ProposeTokenDistribution: {
        return <div className={styles.descriptionCell}>{content}</div>;
      }
      default: {
        return (
          <>
            {renderProposer()}
            {renderDescription()}
            <div className={styles.contentCell}>{content}</div>
          </>
        );
      }
    }
  }
```
so it looks like `renderProposer` won't be called for the `ProposalVariant.ProposeTokenDistribution`.